snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-1

```diff
@@ -0,0 +1,130 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for daily stats ping, tracking instance metrics and settings",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "instance_stats",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "instance_attributes": {
+      "description": "Key-value pairs of instance attributes",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A single instance attribute",
+        "properties": {
+          "key": {
+            "description": "The key for this attribute",
+            "type": "string",
+            "maxLength": 255
+          },
+          "value": {
+            "description": "The value of this attribute",
+            "type": ["string", "boolean", "integer", "null"],
+            "maxLength": 255,
+            "minimum": 0,
+            "maximum": 2147483647
+          }
+        },
+        "required": [
+          "key",
+          "value"
+        ]
+      }
+    },
+    "metrics": {
```

Comment:
Should we add grouped_metrics too?

Reply:
So we don't need to change the schema again to add group_metrics

Reply:
good call, I'll add it to the schema

Reply:
Looking into this some more, we probably want to say that "grouped_metrics" is a required key. So we will have to change this some more. I'll leave in the "grouped_metrics" schema, however once we are sending `grouped_metrics`, I recommend that we alter [snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-1](https://github.com/metabase/metabase/pull/48476/files/4e6afe06fd061a5af249fc7745a76e6ef004c456#diff-830f3618a350a9306a891e1713625bab0c5e8d030fdbced4be15544ee9ed4044) line 159 to be 

```diff 
- "required": ["instance_attributes", "features", "metadata", "metrics"]
+ "required": ["instance_attributes", "features", "metadata", "metrics", "grouped_metrics"]
```

Reply:
Yes. Note that if you make this change after the schema has been deployed, then this means we need to create another schema version. 

We should actually double-check with Joao from SnowcatCloud whether adding new required fields works with a minor version upgrade or whether this would be a major one. If major, we should maybe already add it as required fields and send empty data or something.

Reply:
see: https://github.com/metabase/metabase/pull/48476#issuecomment-2405802467

I decided that it's better to add the "grouped_metrics" schema when we start sending grouped metrics. We would still have to update the schema to make the grouped metrics section required once implemented. This way we can setup the schema, and make it required at the same time.